examples/desk_security_guard.py

```diff
@@ -452,4 +452,3 @@ def did_occur_recently(event_time, max_elapsed_time):
         cozmo.connect_with_tkviewer(run, force_on_top=True)
     except cozmo.ConnectionError as e:
         sys.exit("A connection error occurred: %s" % e)
-
```

Comment:
That's odd, the file still has it on my end. I didn't remove it at all; it was my understanding that all the files were to have a line of empty space at the end of them. Hopefully it'll keep it in this time.